### DISCUSSION TITLE
Unpackaged tests?

### DISCUSSION THREAD

**Discussion by adamplonka** [dotMorten/WinUIEx#26] [General]
Hi,
Are WinUIEx.TestTools supposed to work with unpackaged WinAppSDK 1.0 app? I can't get it working. I'm getting all kinds of errors and not sure if I should keep trying or is it simply not possible at the moment.

Thanks,
Adam

**Comment by dotMorten** [comment]
@plonkaadam Are you able to get any unpackaged WinUI app to work?
See https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/tutorial-unpackaged-deployment?tabs=csharp

**Comment by adamplonka** [comment]
Yes sure. I have an unpackaged app, but beside the models and viewmodels
which I am able to test it also contain couple of custom controls which I
am unable to test currently

**Comment by dotMorten** [comment]
Sorry for dropping the ball on this discussion. Do you mind submitting a bug issue with a small reproducer?
